DataFormats/L1TrackTrigger/interface/TTTrack_TrackWord.h

```diff
@@ -0,0 +1,216 @@
+////////
+//
+// class to store the 96-bit track word produced by the L1 Track Trigger.  Intended to be inherited by L1 TTTrack.
+// packing scheme given below.
+//
+// author: Mike Hildreth
+// date:   April 9, 2019
+//
+///////
+
+#ifndef L1_TRACK_TRIGGER_TRACK_WORD_H
+#define L1_TRACK_TRIGGER_TRACK_WORD_H
+
+#include <iostream>
+#include <string>
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+
+class TTTrack_TrackWord {
+public:
+  // Constructors:
+
+  TTTrack_TrackWord() {}
+  TTTrack_TrackWord(const GlobalVector& Momentum,
+                    const GlobalPoint& POCA,
+                    double theRinv,
+                    double theChi2,
+                    double theBendChi2,
+                    unsigned int theHitPattern,
+                    unsigned int iSpare);
+
+  TTTrack_TrackWord(unsigned int theRinv,
+                    unsigned int phi0,
+                    unsigned int tanl,
+                    unsigned int z0,
+                    unsigned int d0,
+                    unsigned int theChi2,
+                    unsigned int theBendChi2,
+                    unsigned int theHitPattern,
+                    unsigned int iSpare);
+
+  void setTrackWord(const GlobalVector& Momentum,
+                    const GlobalPoint& POCA,
+                    double theRinv,
+                    double theChi2,
+                    double theBendChi2,
+                    unsigned int theHitPattern,
+                    unsigned int iSpare);
+
+  void setTrackWord(unsigned int theRinv,
+                    unsigned int phi0,
+                    unsigned int tanl,
+                    unsigned int z0,
+                    unsigned int d0,
+                    unsigned int theChi2,
+                    unsigned int theBendChi2,
+                    unsigned int theHitPattern,
+                    unsigned int iSpare);
+
+  // getters for unpacked and converted values.
+  // These functions return real numbers converted from the digitized quantities using the LSB defined for each.
+
+  float get_iRinv();
```

Comment:
Coding rule 2.11 (http://cms-sw.github.io/cms_coding_rules.html) discourages the use of "get" in getters

Reply:
Yes, but changing this now breaks the entire L1 Trigger suite while they are working on the TDR.  Will fix later.

Reply:
(Should have pointed out, but you probably know, that none of that code is in a release...)